tests/default.nix

```diff
@@ -28,5 +28,5 @@ import nmt {
     xresources = ./modules/xresources.nix;
   } // pkgs.lib.optionalAttrs pkgs.stdenv.hostPlatform.isLinux {
     i3-keybindings = ./modules/services/window-managers/i3-keybindings.nix;
-  };
+  } // (import ./modules/programs/tmux);
```

Comment:
so instead of importing every test file here, I import a package that itself import the files. IMO less noise in this file.

Reply:
I think that sounds fair enough. But I think the parentheses are superfluous here. Should be enough with `} // import ./modules/programs/tmux;`.